app/controllers/registration_tables_controller.rb

```diff
@@ -84,25 +86,42 @@ def create
     @registration_table = RegistrationTable.new(registration_table_params)
     respond_to do |format|
       if @registration_table.save
-        format.html { redirect_to [@event], notice: 'Table was successfully added.' }
-        format.json { render :show, status: :created, location: [@event] }
+        if @registration_table.payment_ok?
+          format.html {redirect_to [@event], notice: 'Table was successfully added.'}
+        else
+          format.html {redirect_to new_event_session_table_registration_table_table_payment_path(@event, @session, @table, @registration_table), notice: PREMIUM_MESSAGE}
```

Comment:
Is this standard ruby notation for redirections? (I'm just assuming it is, there are similar ones elsewhere). It gets a little hard to read with the word table in there three times (esp. the ...registration_table_table_payment...). Not sure if there's a good way to address this though.